apps/nextjs-website/src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import AdbOutlinedIcon from '@mui/icons-material/AdbOutlined';
+
+type ChatMessageProps = {
+  message: string;
+  sender?: string;
+  timestamp: string;
+};
+
+const ChatMessage = ({ message, sender, timestamp }: ChatMessageProps) => {
+  const { palette } = useTheme();
+  const bgColor = sender ? palette.background.paper : `${palette.info.light}80`;
+  const titleColor = palette.text.primary;
+  const textColor = palette.text.primary;
+  const senderLabel = sender || 'AI ChatBot';
```

Comment:
`AI Chatbot` should be translated

Reply:
remember also to add translation decorator to stories if u fix this line

Reply:
This line will be remove in the next iteration of the component